x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table_groups.tsx

```diff
@@ -281,13 +281,20 @@ export const SpikeAnalysisGroupsTable: FC<SpikeAnalysisTableProps> = ({
                     <br />
                   </>
                 ) : null}
-                {repeatedValues.length > 0 ? (
+                {repeatedValues.length > 0 && valuesBadges.length ? (
                   <FormattedMessage
                     id="xpack.aiops.explainLogRateSpikes.spikeAnalysisTableGroups.moreRepeatedLabel"
                     defaultMessage="+{count, plural, one {# more field/value pair} other {# more field/value pairs}} also appearing in other groups"
                     values={{ count: repeatedValues.length }}
                   />
                 ) : null}
+                {repeatedValues.length > 0 && !valuesBadges.length ? (
+                  <FormattedMessage
+                    id="xpack.aiops.explainLogRateSpikes.spikeAnalysisTableGroups.onlyMoreRepeatedLabel"
+                    defaultMessage="{count, plural, one {# field/value pair} other {# more field/value pairs}} also appearing in other groups"
```

Comment:
You need to drop the `more` from the `other` case too.

Reply:
Oh yes - don't know how I missed that 🙏  - added to the singular case in cf37b208f70a4b41bec146de48b4b1793ebc0d70